src/main/java/com/mate/dao/jdbcimpl/ManufacturerDaoJdbcImpl.java

```diff
@@ -0,0 +1,117 @@
+package com.mate.dao.jdbcimpl;
+
+import com.mate.dao.ManufacturerDao;
+import com.mate.exception.DataProcessingException;
+import com.mate.lib.Dao;
+import com.mate.model.Manufacturer;
+import com.mate.util.ConnectionUtil;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@Dao
+public class ManufacturerDaoJdbcImpl implements ManufacturerDao {
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        String insertQuery = "INSERT INTO manufacturers (name, country)"
+                + "VALUES (?, ?)";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement preparedStatement =
+                        connection.prepareStatement(
+                                insertQuery, Statement.RETURN_GENERATED_KEYS)) {
+            preparedStatement.setString(1, manufacturer.getName());
+            preparedStatement.setString(2, manufacturer.getCountry());
+            preparedStatement.executeUpdate();
+            ResultSet resultSet = preparedStatement.getGeneratedKeys();
+            if (resultSet.next()) {
+                manufacturer.setId(resultSet.getObject(1, Long.class));
+            }
+            return manufacturer;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't get connection in method create", e);
+        }
+    }
+
+    @Override
+    public Optional<Manufacturer> get(Long id) {
+        String selectQuery = "SELECT * FROM manufacturers where id = ?";
+        try (Connection connection = ConnectionUtil.getConnection();
+                 PreparedStatement preparedStatement =
+                         connection.prepareStatement(
+                                 selectQuery, Statement.RETURN_GENERATED_KEYS)) {
+            preparedStatement.setLong(1, id);
+            ResultSet resultSet = preparedStatement.executeQuery();
+            if (!resultSet.next() || resultSet.getBoolean("deleted")) {
+                return Optional.empty();
+            }
+            return Optional.of(parseFromResultSet(resultSet));
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't get connection in method get", e);
```

Comment:
```suggestion
            throw new DataProcessingException("Can't get manufacturer by id: " + id, e);
```
rewrite other messages to make them more informative